### DISCUSSION TITLE
blog/building-a-blog-using-sanity-studio-v3-and-next-js-13-app-router

### DISCUSSION THREAD

**Discussion by ndrvndr** [ndrvndr/personal-website#3] [General]
Learn how to integrate Sanity Studio V3 with Next.js 13 (App Router) to create a stunning blog, following this step-by-step guide.

**Comment by ndrvndr** [comment]
Hello, if you enjoy this article, don't forget to leave a 👍, and if you have any questions or want to discuss something, please leave a comment.

**Reply by Sebbys** [reply]
Thanks for the tutorials 🙏🏻

**Reply by ndrvndr** [reply]
You're welcome👌
